frontend/src/app/shared/components/tabs/scrollable-tabs/scrollable-tabs.component.ts

```diff
@@ -84,7 +92,49 @@ export class ScrollableTabsComponent implements AfterViewInit, OnChanges {
     }
   }
 
-  public onScroll(event:any):void {
+  public startDebouncedTabActivation(tab:TabDefinition):void {
+    // 'dragenter' events are always fired before 'dragleave' events. Hence, when dragging directly from one tab to
+    // another, first the dragenter of the new tab is fired, before we get a dragleave from the old one.
+    // Therefor we keep the drag stack, which can raise from 0 to 2. And we only clear the debounced tab activation
+    // completely when we fully leave tabs (which means, drag stack is 0).
+    this.dragTargetStack += 1;
```

Comment:
I had a map first, but replaced it with the stack because I found it much easier to read :D 

in addition, a map might be overkill, as you have only one cursor dragging along the page. Hence, you should never have more then two items in that map, same thing like the stack cannot grow bigger then 2.